src/pybind/mgr/mgr_module.py

```diff
@@ -210,6 +210,9 @@ def __del__(self):
     def log(self):
         return self._logger
 
+    def audit_log(self, priority, record):
```

Comment:
I suggest omitting the audit_log method from standby modules -- they should never be doing any work that requires audit logging.  StandbyMgrModule only takes a `clog` reference for internally reporting issues loading/executing the module.

Reply:
Done